tests/system/test_dbapi.py

```diff
@@ -1113,44 +1158,13 @@ def test_execute_many(self):
 
         assert res[0] == 1
 
-    @pytest.mark.noautofixt
-    def test_DDL_autocommit(self, shared_instance, dbapi_database):
-        """Check that DDLs in autocommit mode are immediately executed."""
-
-        try:
-            conn = Connection(shared_instance, dbapi_database)
-            conn.autocommit = True
-
-            cur = conn.cursor()
-            cur.execute(
-                """
-                CREATE TABLE Singers (
-                    SingerId     INT64 NOT NULL,
-                    Name    STRING(1024),
-                ) PRIMARY KEY (SingerId)
-            """
-            )
-            conn.close()
-
-            # if previous DDL wasn't committed, the next DROP TABLE
-            # statement will fail with a ProgrammingError
-            conn = Connection(shared_instance, dbapi_database)
-            cur = conn.cursor()
-
-            cur.execute("DROP TABLE Singers")
-            conn.commit()
-        finally:
-            # Delete table
-            table = dbapi_database.table("Singers")
-            if table.exists():
-                op = dbapi_database.update_ddl(["DROP TABLE Singers"])
-                op.result()
-
-    def test_ddl_execute_autocommit_true(self, dbapi_database):
-        """Check that DDL statement in autocommit mode results in successful
-        DDL statement execution for execute method."""
+    @pytest.mark.parametrize("autocommit", [True, False])
+    def test_ddl_execute(self, autocommit, dbapi_database):
+        """Check that DDL statement results in successful execution for execute
+        method in autocommit mode while it's a noop in non-autocommit mode."""
```

Comment:
This comment should include the fact that the connection also uses buffered DDL. And that again means that the DDL statement is not a no-op (I think), but it is being buffered. And the test should then verify tha the statement is actually buffered, and that it is not really a no-op.

Reply:
Done